util/all_tests.json

```diff
@@ -1,49 +1,61 @@
 [
   {
-    "cmd": ["crypto/crypto_test"]
+    "cmd": ["crypto/crypto_test"],
+    "valgrind_supp": ["valgrind_suppressions_crypto_test.supp"]
   },
   {
-    "cmd": ["crypto/crypto_test", "--gtest_also_run_disabled_tests", "--gtest_filter=BNTest.DISABLED_WycheproofPrimality"]
+    "cmd": ["crypto/crypto_test", "--gtest_also_run_disabled_tests", "--gtest_filter=BNTest.DISABLED_WycheproofPrimality"],
+    "skip_valgrind": true
```

Comment:
Nit: it may be more clean and create more space for future by making the path to test config json file configurable? instead of creating new flag.

```
	testCases, err := testconfig.ParseTestConfig("util/all_tests.json")
```

Reply:
Feel free to skip `Nit` comments in this PR.

Reply:
Yes, although it is nice to be able to configure it one please instead of having to deal with many configs (e.g. forgetting to add a test to one of them)